jx_sqlite/query_table.py

```diff
@@ -225,8 +225,10 @@ def query(self, query):
                     parts = set(columns[i])
                     if e.is_groupby and None in parts:
                         allowNulls = True
-                    parts -= {None}
 
+                    if None not in parts:
+                        allowNulls = False
```

Comment:
The only time we can change `allowNulls` is when using `groupby`. `edges` demand all coordinates be represented, including the not null coordinate to catch all the remainder.